percona/controller/pgcluster/controller.go

```diff
@@ -343,6 +343,12 @@ func (r *PGClusterReconciler) addPMMSidecar(ctx context.Context, cr *v2.PerconaP
 		set.Sidecars = append(set.Sidecars, pmm.SidecarContainer(cr))
 	}
 
+	if cr.Spec.PMM.QuerySource == v2.PgStatStatements {
```

Comment:
@egegunes @hors do we need the same check for pgStatMonitor since it is already present?

Basically, we should only enable/disable pgStatStatements?

Reply:
not sure I get the question